packages/did-ethr-resolver/src/implementations/ewSigner.ts

```diff
@@ -12,6 +12,8 @@ type EIP1193ProviderType = providers.ExternalProvider | providers.JsonRpcFetchFu
  * The purpose of the ethers encapsulation is to allow consumers more flexiblity in ethers version.
  */
 export class EwSigner extends Signer {
+  public readonly provider: providers.Provider
```

Comment:
Even without this line, `EwSigner` exposes `provider` because it extends `Signer`. This line is so that compiler can be sure that `provider` is not undefined in `super()` call in `Operator`

Reply:
I disagree that it defeats purpose of `EwSigner`. It is still convenient for folks to be able to instantiate `EwSigner` without needing an `ethers` type. I agree that `Resolver` shouldn't accept an `ethers` provider in constructor, I think we can fix in a separate issue.